fbfmaproom/data-conversion-scripts/oromia/bad_years.csv

```diff
@@ -0,0 +1,9 @@
+one  column for the years (year); a column for each season (in lowercase); the values in the season represent a rank.
+year,ond
+2000,6
+2003,5
+2008,3
+2011,2
+2015,4
+2021,1
+2022,7

```

Comment:
You added the oromia csv file but not the djibouti one(s)?

Reply:
Oh, actually we usually don't keep the data files in git. This is a tiny one, so there's no technical problem keeping it in git, but this isn't the place I would normally look for it.